zmon-controller-ui/js/controllers/CheckDefinitionEditCtrl.js

```diff
@@ -45,13 +45,17 @@ angular.module('zmon2App').controller('CheckDefinitionEditCtrl', ['$scope', '$ro
         // for route '/check-definitions/edit/:checkId' [edit existing check]
         $scope.checkId = $routeParams.checkId;
 
+        $scope.whitelistedForInterval = window.zmonBootData.minCheckInterval.whitelistedChecks.indexOf(parseInt($scope.checkId)) >= 0;
```

Comment:
This will break in case `minCheckInterval` is not defined on `zmonBootData`.

Instead of using those variables straight from `window` object within the controller, it would be better to have some configuration service  which would set default values and overwrite them with content of `zmonBootData`. so that you don't have to add if conditions to make sure you are not trying to access fields from `undefined` objects when using the configs.

something like `ConfigSrv`.

Reply:
would eventually be better for testing as well, because enabling to better mock configuration